_posts/2021-03-03-cockpit-239.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 239
+author: lis
+date: 2021-03-03
+tags: cockpit linux
+slug: cockpit-239
+category: release
+summary: Cockpit with restyled updates page
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).  We release regularly.
+
+Here are the release notes from Cockpit version 239.
+
+### Machines: Allow editing disk cache mode
```

Comment:
Probably one of these headline adjustments

```suggestion
### Machines: Disk cache mode editing
```

```suggestion
### Machines: Disk cache mode is now editable
```

Reply:
I prefer to have the headings be a verb describing what changed.